### PR TITLE
Join leave integration

### PR DESCRIPTION
**Pull request by stephend017**
integrated these functions, unable to test because no communities are created yet.

### CODE REVIEW

**Review by smit3164 — COMMENTED** [review]
looks good

**Review by DanielKambich — CHANGES_REQUESTED** [review]
Looks like there is an issue with the build. If you can fix the missing variable then I can approve

File: src/components/communities/CreateCommunityPage.tsx [hunk 1]
```diff
@@ -1,162 +1,165 @@
...
-  codeOfConduct: string;
-};
-
-const formErrors = {
-  name: "Community name cannot be empty",
-  description: "Description cannot be empty",
-  codeOfConduct: "Code of Conduct cannot be empty",
-};
-
-export default function CreateCommunityPage({}: CreateCommunityPageProps) {
```
Comment (smit3164) [inline src/components/communities/CreateCommunityPage.tsx:57]: make sure to put the route/navigation props in here

**Review by DanielKambich — APPROVED** [review]

**Review by smit3164 — APPROVED** [review]

**Comment by DanielKambich** [comment]
Looks like build is fixed now. Will merge it

**Pull request merged**